### DISCUSSION TITLE
Esphome LXC Permission denied

### DISCUSSION THREAD

**Discussion by DNASystem** [tteck/Proxmox#1021] [Help Needed]
Hi I have a problem about Esphome LXC after installing "pip install pillow" because it is a component that allows to operate a display, esphome no longer writes on modules it is giving me this problem how can I solve "permission denied", I don't know if it is caused by this but I can no longer update all the modules, what can I do to fix it?

INFO Reading configuration /root/config/modulo-luci-expander.yaml...
INFO Generating C++ source...
INFO Compiling app...
Processing modulo-luci-expander (board: esp32doit-devkit-v1; framework: arduino; platform: platformio/espressif32 @ 5.2.0)
--------------------------------------------------------------------------------
HARDWARE: ESP32 240MHz, 320KB RAM, 4MB Flash
 - toolchain-xtensa-esp32 @ 8.4.0+2021r2-patch3
LDF: Library Dependency Finder -> https://bit.ly/configure-pio-ldf
Dependency Graph
|-- AsyncTCP-esphome @ 1.2.2
|-- WiFi @ 2.0.0
|-- FS @ 2.0.0
|-- Update @ 2.0.0
|-- ESPAsyncWebServer-esphome @ 2.1.0
|   |-- AsyncTCP-esphome @ 1.2.2
|-- DNSServer @ 2.0.0
|-- ESPmDNS @ 2.0.0
|-- noise-c @ 0.1.4
|   |-- libsodium @ 1.10018.1
|-- Wire @ 2.0.0
|-- ArduinoJson @ 6.18.5
Compiling .pioenvs/modulo-luci-expander/src/esphome/components/api/api_connection.cpp.o
**sh: 1: xtensa-esp32-elf-g++: Permission denied**
*** [.pioenvs/modulo-luci-expander/src/esphome/components/api/api_connection.cpp.o] Error 126
Compiling .pioenvs/modulo-luci-expander/src/esphome/components/api/api_frame_helper.cpp.o
**sh: 1: xtensa-esp32-elf-g++: Permission denied**
*** [.pioenvs/modulo-luci-expander/src/esphome/components/api/api_frame_helper.cpp.o] Error 126
========================== [FAILED] Took 1.29 seconds ==========================

**Comment by tteck** [comment]
My initial thought is to retrieve a backup.
